bundles/org.eclipse.equinox.security.ui/META-INF/MANIFEST.MF

```diff
@@ -15,7 +15,7 @@ Import-Package: javax.crypto.spec,
  org.eclipse.osgi.util;version="[1.1.0,2.0.0)",
  org.osgi.framework,
  org.osgi.util.tracker;version="[1.3.3,2.0.0)"
-Require-Bundle: org.eclipse.equinox.security;bundle-version="[1.0.0,2.0.0)",
+Require-Bundle: org.eclipse.equinox.security;bundle-version="[1.3.1000,2.0.0)",
```

Comment:
Shouldn't put the micro version in the range here.  Typically only go to the minor version `1.3` here.